web/viewer.css

```diff
@@ -766,10 +796,11 @@ select {
 }
 
 .dropdownToolbarButton {
-  width: var(--scale-select-container-width);
+  width: calc(var(--scale-select-container-width) + 2px);
```

Comment:
Do we need the extra `2px` here, and below, or will things perhaps work well enough anyway in forced-colors mode (since the JS-code which measures the necessary size should already leave enough room)?